Bluetooth Troubleshooting Updated Content

HappyGelo05 opened the pull request:
I updated a few positive descriptions for clarity. The previous version included explanations on troubleshooting but didn’t include actual steps, so I added those. I also removed installation commands referencing older Pop!_OS versions (21.10 and 20.04), updated the TLP configuration path from /etc/default/tlp to /etc/tlp.conf, and added Blueman as the Bluetooth manager. Additionally, I introduced bluetoothctl as it allows for efficient hardware device checks and generates detailed logs, making troubleshooting and analysis much easier. Additionally, I added new photos to enhance the article’s visual appeal and provide clearer, more informative context.

ahoneybun committed:
New Fan Noise article (#1171)

* Init work on Fan Noise article

* Fix lint issues

* Fix lint issues

* Update fan-noise.md

Worked on the language

* Update fan-noise.md

...and additional language updates

* add image and fix lint issue

* Address feedback

* Update list for newer and older models

* Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update linux gaming (#1175)

* Update image to new Steam UI

* Update search screenshot and activities to application for 22.04

* Update screenshots

* Update wording

* Move using steam section

* remove note about GameHub as it works now.

* Update linux-gaming.md

fix feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

thomas-zimmerman committed:
Update password.md

updates to the LUKS commands.

Commit:
Fix lint issue

ahoneybun committed:
Merge branch 'master' into update-passwords

thomas-zimmerman committed:
Add note to use launch cables for firmware updates (#1177)

* Add note to use launch cables for firmware updates

Update launch_2-firmware-update.md

* Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update dock article (#1178)

* Update images and renaming article

* Remove Ubuntu 16.04 section as it is no longer supported

* Add back Dash to Dock

* Add content

* Update images and remove old ones

* Rename article to fix link

* Fix lint issue

* Fix lint issue and address feedback

* Remove Dash to Dock installation and instead use dconf editor

* Correct position of the default dock for Ubuntu and Pop!_OS and adjust other content

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <thomas@system76.com>

thomas-zimmerman committed:
Log in from live disk (#1180)

* Update login-from-live-disk.md

Added context to encryption section

* Add some additional comments on what the chroot commands are doing and fixing lint issues

* Address feedback

---------

Co-authored-by: Sara Mildenstein <115432964+sararocks@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Fix shortcuts and address feedback

ahoneybun committed:
Change table to command to check hardware (#1182)

burden committed:
remove gear up promo (#1188)

* remove gear up promo

* update omnibar

Spongecade committed:
Update Minecraft wiki link to new domain (#1187)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

mapachurro committed:
Remove duplicate instruction (#1189)

* Remove duplicate instruction

The instruction regarding taking a picture or screenshot of the instructions showed up twice. I left the later one, as I feel it provided better emphasis to an important point.

* Update system-firmware.md

* Update system-firmware.md

* Update system-firmware.md

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add Boot Menu from the thelio-r3 board (#1183)

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Add meer7 and darp9 (#1185)

* Add meer7

* Add darp9, meer7 and fix lint issue

---------

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Update screenshots and content for 22.04 (#1181)

* Update screenshots and content for 22.04

* Remove Beta line and fix grammar

---------

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Add note about the pang12/13 switch (#1179)

* Add note about the pang12/13 switch

* webcam: Reword article to better incorporate pang12/13 switch

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Francis Smith <fs1995@ymail.com>

Socialhappiness committed:
Create Desktop_zoom.md (#1165)

* Create Desktop_zoom.md

adding an article specifically for making desktop bigger

* work on fixing lint issues

* Rename file

* More lint fixes

* Fix lint issues

* Fix numbered list issue

* Another possible lint fix

* Update desktop-zoom.md

* Add lint rule to keep numbered list with images working

* Update desktop-zoom.md

* Add images

* Fix image links

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Update ubuntu articles (#1176)

* Add note about GNOME Shell extension for newer then Ubuntu 22.04

* Link to boot menu article for up to date steps and address feedback

* Address feedback about note

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Francis Smith <fs1995@ymail.com>

JoseJacsan committed:
Added Shotcut video editor description in list (#1191)

* Added Shotcut video editor description in list

* Added spelling error correction

* Add additional descriptors from the project's homepage

* Update media-production-on-pop.md

Fix lint issue

* fix (lint): Remove double line break at end of file

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

adamdicarlo0 committed:
Fix misplaced word and use "back up" (verb) instead of "backup" (#1193)

burden committed:
Update omnibar with link to Tuxmas page (#1194)

burden committed:
tuxmas sale week 2, update components lib + omnibar copy (#1195)

burden committed:
tuxmas sale week 3 (#1196)

garrettjwilke committed:
bash basics tutorial (#1197)

* bash basics tutorial

* Address feedback

* Address feedback

* Rewrite adding functionality section

---------

Co-authored-by: Garrett Wilke <garrett@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: leviport <levi@system76.com>

ahoneybun committed:
Update images and comment out CLI option as does not support that in v4 yet

Fix lint

thomas-zimmerman committed:
Merge pull request #1200 from system76/update-protonvpn-killswitch

Update images and comment out CLI option

thomas-zimmerman committed:
Merge branch 'master' into update-passwords

thomas-zimmerman committed:
Merge pull request #1166 from system76/update-passwords

Work on adding the advanced luks article to the password article

memachado committed:
Update desktop-environment.md (#1190)

* Update desktop-environment.md

This addition offers users a more comprehensive solution for removing KDE and its associated packages. This enhancement aims to provide clearer and more user-friendly instructions for managing desktop environments on Pop!_OS.

* Split regex from normal autoremove and add explanation

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>

macifell committed:
tuxmas phase 4

macifell committed:
Merge pull request #1205 from system76/tuxmas4

tuxmas phase 4

ahoneybun committed:
Add meer8 to guide and windows articles

bbarker committed:
Update difference-between-pop-ubuntu.md (#1209)

Fix typo in keyboard shortcut for switching workspaces

macifell committed:
Remove tuxmas

macifell committed:
Merge branch 'master' into remove-tuxmas

macifell committed:
Merge pull request #1211 from system76/remove-tuxmas

Remove tuxmas

seansawyer committed:
Update guides.md to include Lemur Pro 12 (lemp12) tech doc (#1212)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

PeterGrand committed:
Use recursive binding before chroot (#1204)

By using -R instead of -B, we ensure that the nested mount of /sys/firmware/efi/efivars/ becomes mounted in our target filesystem, which allows us to do various boot repairs with an EFI system.

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

samcodee committed:
Update article about Differences between Pop!_OS and Ubuntu (#1203)

* Add updated intel nvidia

* Fix typo: "Backported"

* Add new image for download screen

also added info about rpi 4

* Rename intel_nvidia.png to intel-nvidia-rpi.png

* Delete old image

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add pang13 and pang14 to missing spots in articles (#1213)

ahoneybun committed:
Add nix-shell file to use nix for testing

ahoneybun committed:
Update shell.nix

remove shellHooks

ahoneybun committed:
add launch_3 and launch_heavy_3 tech doc links

jklgrasso committed:
Update hardware-failure.md

Changed the gpu-burn example to use tc instead of d to create more of a workload when testing.

jklgrasso committed:
Update hardware-failure.md

Adjusted for RTX GPUs to use -tc and removed the -d switch for GTX GPUS.

thomas-zimmerman committed:
Merge pull request #1221 from system76/gpu-burn-d-to-tc

Update hardware-failure.md

ahoneybun committed:
Fix typo for Packages

thomas-zimmerman committed:
Merge pull request #1207 from system76/add-meer8

Add meer8

thomas-zimmerman committed:
Merge branch 'master' into add-nix-shell

thomas-zimmerman committed:
Merge branch 'master' into update-launch

thomas-zimmerman committed:
Merge pull request #1219 from system76/update-launch

add launch_3 and launch_heavy_3 tech doc links

thomas-zimmerman committed:
Merge branch 'master' into add-nix-shell

thomas-zimmerman committed:
Merge pull request #1216 from system76/add-nix-shell

Add nix-shell file to use nix for testing

ahoneybun committed:
Clean up and remove old install Ubuntu images for install-ubuntu article

ahoneybun committed:
Update/Move content and update screenshots

ahoneybun committed:
Add more images

ahoneybun committed:
Add Software Updater screenshots

ahoneybun committed:
Remove old screenshots

ahoneybun committed:
update: clean up the windows table and add missing systems (#1214)

* update: clean up the windows table and add missing systems

* fix: typo and lint issue

* Update windows.md

Remove left over from merge conflict

---------

Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update deploy.yml (#1225)

* Update deploy.yml

* Update ci.yml

* Update deploy.yml

macifell committed:
Revert "Update deploy.yml (#1225)"

This reverts commit 5427233722b0a763646de0ca615ef10c81c59432.

thomas-zimmerman committed:
Merge pull request #1227 from system76/revert-1225-update-workflow

Revert "Update deploy.yml"

ahoneybun committed:
Finish up article with initial setup screenshots

leviport committed:
Merge branch 'master' into update-ubuntu-24.04

ahoneybun committed:
Add images for 23.10 to 24.04 upgrade

leviport committed:
Merge branch 'master' into update-upgrade-ubuntu

ahoneybun committed:
Update Ubuntu supported release and links for making USB while removing DVD reference

ahoneybun committed:
Address the rest of the feedback minus two from the review

ahoneybun committed:
Update GRUB and LUKS/Plymouth screenshots

ahoneybun committed:
Cleaned up remaining fixes in the review

ahoneybun committed:
Updates the wording about third-party software and media support selection in the installer

ahoneybun committed:
Address feedback

jacobgkau committed:
fix: Grammar & typos

jacobgkau committed:
fix: Additional ordering, grammar, and consistency tweaks

thomas-zimmerman committed:
Merge pull request #1228 from system76/update-ubuntu-24.04

Update install-ubuntu article for 24.04

ryanbaekr committed:
Add gnome-shell-extensions to the packages needed for Ubuntu (#1232)

* Add gnome-shell-extensions to the packages needed for Ubuntu

Ubuntu 24.04 doesn't ship with gnome-shell-extensions

* Update pop-shell.md

* improv: List reason for installing extension-prefs package

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

taivlam committed:
Fix typo in launch-keyboard.md (#1234)

Use "its" (possessive), instead of "it's" (contraction)

taivlam committed:
Fix typo in launch_2-firmware-update.md (#1233)

Remove extra "for"

Co-authored-by: Levi Portenier <levi@system76.com>

jacobgkau committed:
Add ROCm article (#1231)

ahoneybun committed:
Merge branch 'master' into fix-typo

Commit:
Merge pull request #1223 from system76/fix-typo

Fix typo for Packages

ahoneybun committed:
Update model list and remove unsupported OSes (#1224)

ahoneybun committed:
Redo bootloader (#1198)

* Reorder article and update sections

* Update notes about BIOS installs

* Fix header for encrypted disk

* Add Pop!_OS GRUB steps and fix lint issue

* Address feedback

* Update bootloader.md

Removed > on line 85 to (hopefully) fix markdown failure.

---------

Co-authored-by: brandonbaez7 <160044194+brandonbaez7@users.noreply.github.com>

trenner1 committed:
Update pop-keyboard-shortcuts.md

add the key strokes for moving active windows between upper and lower displays

jacobgkau committed:
Add recent tech docs models & link to tech docs homepage (#1237)

thomas-zimmerman committed:
add note for EC update (#1239)

* add note for EC update

Firmware updates that include EC firmware updates require a full power cycle to complete. Added a note suggesting either AC power unplug/plug or holding power button for 15 seconds.

* Update system-firmware.md

..and fix language.

Commit:
Update upgrade-ubuntu.md

Removed documentation referring to DVDs. Changed documentation referring to the Activities Overview, which now refers to the Applications menu and the Show Apps button. Highlighted that the Software Updater may take a moment to show the settings page.

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

ahoneybun committed:
Update upgrade-ubuntu.md

fix: lint issue

spaceoden committed:
Add note about need to disable ppd on NixOS (#1240)

* Add Nixos power note to system76-software.md

ppd (enabled by default on Gnome NixOS installs) must be disabled for system76-power to start. Include that info in docs.

* Update system76-software.md

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

ws909 committed:
Update the steps to generate a log file in freezing-behavior.md (#1241)

* Update the steps to generate a log file in freezing-behavior.md

The existing explanation was for a previous version of the system.

* Update freezing-behavior.md

Readd the log file creation in the System76 Driver in addition to in the Settings application

* Update freezing-behavior.md

fix Pop!_OS

* Update freezing-behavior.md

* Update freezing-behavior.md

fix lint

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

UrielCh committed:
Update cuda.md, replace cuda 12.2 with 12.4 (#1238)

* Update cuda.md, replace cuda 12.2 by 12.5

Cuda 12.2 is deprecated, and display nasty warning.

the driver version 555 is currently 6 day old.

the driver version 550 was built on top of cuda 12.4.

* replace cuda 12.5 by 12.4

---------

Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

jessecambon committed:
Add info on Docker usage to ROCm install instructions (#1242)

* Added info on docker usage to rocm.md

See comments here for more information https://gist.github.com/guilt/6c901f7ac0a726685b6334798da77c00?permalink_comment_id=5131866#gistcomment-5131866

* adjusted docker terminology in rocm.md

* Add docker site link to rocm.md

* Minor grammar corrections

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

itsfaizankhan committed:
Fixed typo in switch-from-macos-to-popos.md

jacobgkau committed:
improv (pop-key-shortcuts): Improve wording & symbol consistency

esplinr committed:
Update system-firmware.md (#1243)

* Update system-firmware.md

Make clear what happens if the firmware updater is run with external devices plugged in.

* Update system-firmware.md

Incorporating feedback

* enhance (firmware): Split additional info into sub-bullets

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

jacobgkau committed:
bluetooth: Add btusb reload commands (from #1217) (#1246)

jacobgkau committed:
bootloader: Make bind mounts recursive for EFI vars support

nonergodic committed:
Add bullet point to Before You Begin section in system-firmware doc (#1134)

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>

jacobgkau committed:
Codecs update (#1247)

* Install libdvd-pkg

* codecs: Make command instructions consistent & include TUI prompt info

---------

Co-authored-by: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Co-authored-by: Levi Portenier <levi@system76.com>

burden committed:
Update banner image (#1249)

ahoneybun committed:
Fix lint

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

ahoneybun committed:
Update upgrade-ubuntu.md

fix lint

jacobgkau committed:
fix: Minor grammar/typo corrections

jackpot51 committed:
Do not keep gh-pages history

Keeping the history of gh-pages adds a lot of bloat when trying to clone the repository. The force_orphan option added pushes just one commit with no prior history to the gh-pages branch on every change, which should significantly reduce the size of the repository.

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

thomas-zimmerman committed:
Merge pull request #1229 from system76/update-upgrade-ubuntu

Update upgrade-ubuntu article for 24.04

jacobgkau committed:
List Pop!_Shell branches for various Ubuntu versions (#1252)

* shell: List branches for different Ubuntu versions

* shell: Other minor grammar fixes

* shell: Mark code block type

jacobgkau committed:
Add article: VirtualBox Black Screen on Threadripper (#1256)

Nested paging must be disabled on some Threadripper and Threadripper Pro
products in order for VirtualBox guests to display properly.

I compressed the larger screenshots with WebP, but left the smaller ones
as optimized PNG because the compression looks bad when they're scaled up.

ahoneybun committed:
fix dmidecode typo (#1254)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Levi Portenier <levi@system76.com>

ahoneybun committed:
Update ubuntu-basics article for 24.04 (#1230)

* Remove old 16.04 screenshots

* Add Ubuntu 24.04 screenshots

* Fix typo and add more screenshots

* Remove older 21.04 and below iamges and update 24.04 images

* Update wording about adding to the launcher

* Ubuntu Basics: Minor style tweaks

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Levi Portenier <levi@system76.com>

mlundyCO committed:
Update system-firmware.md (#1251)

Moved sentences about EFI to appropriate section.

Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

basujindal committed:
Update graphics-switch-ubuntu.md (#1222)

* Update graphics-switch-ubuntu.md

Added note.

* Update graphics-switch-ubuntu.md

Add another note

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Levi Portenier <levi@system76.com>

ahoneybun committed:
Add photos transfer article (#1210)

* Init work on new article

* Continue work on article adding more sections and images

* Work on wording

* Add Pixel/Android steps and more images/text

* address feedback

* update text

* Verbiage tweaks in photo transfer article.

---------

Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

burden committed:
update footer (#1258)

ahoneybun committed:
Remove accessories (#1259)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Update windows.md

add addw4 to the table

Commit:
updates link to the guide article rather than per model and adds photo for the v2 board's power button

thomas-zimmerman committed:
Merge pull request #1266 from system76/update-power-on-failure

Updating the page to be more clear and include Thelio Io v2

Commit:
change note about the Pangolins to a table

Commit:
add pang15 to the list of models

Commit:
update table based on feedback

Commit:
address feedback

thomas-zimmerman committed:
Merge pull request #1268 from system76/update-battery-thresholds-article

Update battery thresholds article

leviport committed:
Merge branch 'master' into add-addw4-windows

thomas-zimmerman committed:
Merge pull request #1260 from system76/add-addw4-windows

Update windows.md

leviport committed:
Add meer9 to open firmware & EC list, and fix table whitespace (#1271)

* Add meer9 to open firmware & EC list, and fix table whitespace

* List meer9 EC as N/A (it has no EC, open or otherwise)

thomas-zimmerman committed:
Add a new support document as a landing page for URLs in old support … (#1264)

* Add a new support document as a landing page for URLs in old support messages to allow both an explnation
and quick way to open support ticket if they still need help.

* Address some lint issues with spaces.

* respond and update language from jacobgkau review

* update for AR language

* Address Mikes updates

* address points raised that github UI was hiding.

* Additional link & grammar tweaks

* Remove unnecessary footer & additional link tweak

* Additional link tweaks & punctuation fix

* Additional clarifications & formatting tweaks

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Levi Portenier <levi@system76.com>

thomas-zimmerman committed:
Update system76-driver.md (#1274)

Remove the `apt update` after adding the PPA and add a `apt upgrade` afterwords. Adding the PPA will always get updated release files. Need the upgrade to upgrade packages in the PPA.

thomas-zimmerman committed:
Update install-ubuntu.md (#1275)

* Update install-ubuntu.md

Add a "apt upgrade" after the PPA is added.

* Update install-ubuntu.md

remove apt update after adding the PPA as it runs by default now.

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Levi Portenier <levi@system76.com>

Commit:
move battery article as it is now incorrect for 22.04 and above

ahoneybun committed:
update action/cachev2 to v4 to fix build issue (#1278)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

leviport committed:
Clarify hibernation disclaimer sentence (#1273)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Merge branch 'master' into fix-issue-1201

thomas-zimmerman committed:
Merge pull request #1277 from system76/fix-issue-1201

move battery article as it is now incorrect for 22.04 and above

raxod502 committed:
Add tips about getting gpu-burn to compile

jacobgkau committed:
Switch CUDA package & slight clarifications

thomas-zimmerman committed:
Merge pull request #1279 from raxod502/patch-1

Add tips about getting gpu-burn to compile

kspeakman committed:
note on etcher data collection

burden committed:
publish web infra page

jacobgkau committed:
fix: Put title in title case

thomas-zimmerman committed:
Merge pull request #1286 from system76/visible-infra-page

publish web infra page

thomas-zimmerman committed:
Update deploy.yml

Update actions/cache to v4 to fix deploy failures on github.
Should fix deploy error on [PR#1286](https://github.com/system76/docs/pull/1286)

AlexisSweetMcKay committed:
Update boot-menu.md

Added link to open firmware models page for convenience

AlexisSweetMcKay committed:
Update boot-menu.md

Amended ! error

AlexisSweetMcKay committed:
Update boot-menu.md

Changed link to open firmware models text in chart

EricMTodd committed:
Update boot-menu.md

Added a catch-all statement instructing users to try all the FX keys if the ones listed on the boot-splash or in this article don't work.

Changed BIOS to BIOS/UEFI.

EricMTodd committed:
Update boot-menu.md

Made grammatical and punctuation adjustments suggested by @jacobgkau

thomas-zimmerman committed:
Merge pull request #1289 from system76/EricMTodd-patch-1

Update boot-menu.md

thomas-zimmerman committed:
Update boot-menu.md

Fix the link.

thomas-zimmerman committed:
Merge branch 'master' into AlexisSweetMcKay-patch-1

thomas-zimmerman committed:
Merge pull request #1288 from system76/AlexisSweetMcKay-patch-1

Update boot-menu.md

wackbyte committed:
Add meer8, previous models, and meer10 to open firmware list

wackbyte committed:
Add meer9 to tech docs link list

wackbyte committed:
Fix formatting of "Pop!_OS" in article description

wackbyte committed:
Fix typo

wackbyte committed:
Re-add CC BY-SA 4.0 license (#1297)

wackbyte committed:
Update Luanti/Minetest article (#1295)

- Minetest rebranded as Luanti ([blog post](https://blog.luanti.org/2024/10/13/Introducing-Our-New-Name/)).
- It would be more accurate to refer to Luanti itself as an "engine" rather than a "game"--[their home page](https://www.luanti.org/) also calls it an "engine".
- It can now be [installed from a PPA](https://www.luanti.org/downloads/), simplifying the installation process greatly.
- Content is now available from [ContentDB](https://content.luanti.org/), which is accessible in-game, further simplifying the installation process.
- Multiplayer still works the same, so there was no need to update that section.

Co-authored-by: Levi Portenier <levi@system76.com>

btkostner committed:
chore(ci): update github actions to latest version

kuchtact committed:
Fix incorrect usage of effect instead of affect (#1302)

EricMTodd committed:
Update flash-firmware-from-usb.md (#1303)

Fixed a small typo on line 65 that read "coping the file(s)" to "copying the file(s)".

Co-authored-by: Levi Portenier <levi@system76.com>

EricMTodd committed:
Update terminal-basics.md

Added a missing period after `ls` on line 47. Added additional information detailing `cd` by itself will return the user to the home directory.

EricMTodd committed:
Merge pull request #1305 from system76/EricMTodd-patch-1

Update terminal-basics.md

thomas-zimmerman committed:
Merge pull request #1292 from wackbyte/meer-open-firmware

Add meer8, previous models, and meer10 to open firmware list

jacobgkau committed:
chore(ci): Bump Nix Shell's Node.js from 18 to 20

thomas-zimmerman committed:
Merge pull request #1306 from system76/nix-node

chore(ci): Bump Nix Shell's Node.js from 18 to 20

jacobgkau committed:
Add new dependency to DaVinci Resolve installation

Also reorder dependencies to be in alphabetical order

thomas-zimmerman committed:
Merge pull request #1308 from system76/resolve-libglu

Add new dependency to DaVinci Resolve installation

WatchMkr committed:
Create pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Update pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Update pop-cosmic-keyboard-shortcuts.md

jacobgkau committed:
Add another dependency for DaVinci Resolve installation

WatchMkr committed:
Typo fix pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Add stacking shortcut pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Fix typo pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Add missing ) pop-cosmic-keyboard-shortcuts.md

thomas-zimmerman committed:
Merge branch 'master' into resolve-wayland

jacobgkau committed:
Fix spacing lints in pop-cosmic-keyboard-shortcuts

thomas-zimmerman committed:
Merge pull request #1310 from system76/resolve-wayland

Add another dependency for DaVinci Resolve installation

GeorgeAverkin committed:
Update pop-cosmic-keyboard-shortcuts.md (#1311)

Fix "Switch between added language/input source" shortcut

jacobgkau committed:
Document oryp5's Open Firmware transition

thomas-zimmerman committed:
Merge pull request #1313 from system76/oryp5-transition

Document oryp5's Open Firmware transition

jacobgkau committed:
Homepage tweaks

thomas-zimmerman committed:
Merge pull request #1314 from system76/updates

Homepage tweaks

Socialhappiness committed:
Update fan-noise.md

Adding Mechanical Troubleshooting Section to put more fan noise troubleshooting into one place

Socialhappiness committed:
Update fan-noise.md

corrected duplicate section

jacobgkau committed:
Correct header hierarchy & small punctuation tweaks

thomas-zimmerman committed:
Merge pull request #1318 from system76/Socialhappiness-patch-4

Update fan-noise.md

Socialhappiness committed:
Make article more general & add warning about losing logins (#1300)

Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Levi Portenier <levi@system76.com>

thomas-zimmerman committed:
Update log-files.md

Get manual log creation closer to what pop-support does.

thomas-zimmerman committed:
Merge pull request #1328 from system76/thomas-zimmerman-patch-1

Update log-files.md

HappyGelo05 committed:
New folder for bluetooth images

HappyGelo05 committed:
Updated bluetooth content

jacobgkau committed:
Clarify support tickets are only for customers

jacobgkau committed:
Remove note markers

HappyGelo05 committed:
Remove note blocks

Socialhappiness commented on content/bluetooth.md:
````diff
@@ -15,15 +15,42 @@ section: network-troubleshooting
+version
+```
+
+![bluetoothclt version](/images/bluetooth/bluetooth_1.png)
````
bluetoothclt is mispelled

Socialhappiness commented on content/bluetooth.md:
```diff
@@ -15,15 +15,42 @@ section: network-troubleshooting
+
 ### Signal Interference
 
 Bluetooth uses the same bandwidth as the 2.4Ghz Wi-Fi band, and in most of our machines it is on the same chip as the Wi-Fi module. They usually have two antennae, one for Bluetooth, and one for Wi-Fi, but it is possible for other Wi-Fi or Bluetooth devices signals to cross and to cause connection issues. If users are in an area crowded with other Wi-Fi networks or devices, the interference from  ...
```
Just changed the wording a little bit:
Bluetooth uses the same bandwidth as the 2.4Ghz Wi-Fi band, and in some machines it is on the same chip as the Wi-Fi module. Th Wi-Fi module usually has two antennae, one for Bluetooth, and one for Wi-Fi, but it is possible for other Wi-Fi or Bluetooth devices signals to cross and to cause connection issues. If users are in an area crowded with other Wi-Fi networks or devices, the interference from these outside sources can impact performance and range.

Socialhappiness commented on content/bluetooth.md:
```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
 
 Bluetooth issues can be troubleshooted in several ways.  The first thing to check is toggling airplane mode which will sometimes get Bluetooth functioning again.  Next, make sure Bluetooth is enabled in the top bar, or in the <u>Bluetooth</u> system settings.
 
```
After the airplane mode, I suggest adding a new line and a screenshot for this section:
Confirm Bluetooth is enabled in the top bar, or in the <u>Bluetooth</u> system settings.

Socialhappiness commented on content/bluetooth.md:
```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
-
-*For Pop!\_OS 22.04 or higher:*
+Installing Bluetooth related software with this command:
 
```
Install should be here instead of Installing.

Socialhappiness commented on content/bluetooth.md:
```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
+
+![bluetooth status systemd](/images/bluetooth/bluetooth_5.png)
+
 Then, run <u>Bluetooth Manager</u>. Check for the device being trusted, and also try re-pairing in that program.
```
would a screenshot or more detailed instructions be better here?

Socialhappiness commented on content/bluetooth.md:
```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
+
+![bluetooothctl device connected](/images/bluetooth/bluetooth_9.png)
+
 ### Useful Commands
```
Can 'useful commands' section go to the beginning of the article after the how to remove and pair instructions?

Socialhappiness reviewed (changes requested):
Great job! I suggested to add basic instructions in the beginning and to remove/reword a few things. Please review. Thank you!

HappyGelo05 committed:
Added new photos

HappyGelo05 committed:
Remove unnecessary items, correct and adjust sections, and add new photos

HappyGelo05 committed:
fix lint errors

Socialhappiness reviewed (changes requested):
Almost there! A few more changes requested. And can we add a section on how to get in touch with System76 support at the end?
For example:
If you purchased a computer from System76 and you are still encountering bluetooth problems, please have your serial number ready and [open a support ticket](https://system76.com/contact/support).

HappyGelo05 committed:
Fix button formats and update Bluetooth description

HappyGelo05 committed:
update bluetooth_2 image

Socialhappiness committed:
Support Page Reorganization (#1339)

* Reorganizes the page to be more Pop and System76 hardware prominent with an attempt to make it easier to find things with some title and description updates, and specified 22.04 vs 24.04 in some places, removed the sections in the two boxes at the top so they are even and contain the most important information like tech docs and opening a support ticket.

* lint errors and reworded sections for Hardware and Software and the top card title

* Tech docs card tweaks

---------

Co-authored-by: emma76 <emma76@pop-os.local>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

Socialhappiness committed:
Update install 2404 articles (#1340)

* updated pop 24.04 images and removed the pop shop mention for nvidia

* Remove duplicate images from Markdown content folder

* Tweak wording to match current download page design

---------

Co-authored-by: emma76 <emma76@pop-os.local>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

marwinpinlac-web committed:
Update on Audio Troubleshooting Guide (#1336)

* Add audio screenshots

* Applied changes in audio.md, added images necessarry for the update and removed images that are not in use.

* fixed lint error, reordered and made updates on audio.md

* Updated file name of images and updated content of audio md following the comments on the recent review

* Minor phrasing & formatting tweaks

* Updated screenshots to COSMIC

* New commit for Lint error

* Compress large PNG screenshots into JPGs

* Optimize remaining PNGs

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

jacobgkau committed:
chore(ci): update github actions to latest version (again)

ScatteredComet committed:
Update ROCm instructions to reference the latest version (#1338)

* Update rocm.md

fix issue #3670

* Make instructions work on 24.04 & fix silent password prompt

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

Socialhappiness committed:
updating iso filenames with wildcard and remove verbose and repeated … (#1342)

* updating iso filenames with wildcard and remove verbose and repeated words, added 2404 installer launch window

* Expand link text & remove inconsistent version number

* Additional note, link, and other copy revisions

* Update iso checksum screenshot

* Update Disks screenshots & copy

* Optimize all live disk PNG images

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

jacobgkau committed:
Minor extra-drive copy updates to match latest screenshot

Socialhappiness committed:
Update default apps (#1345)

* Removed existing text about default app inclusion that was written in 2021 and added the actual default applications and screenshots, as well as instructions for accessing and changing default apps.

* Slight reorganization and some text revisions

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

Socialhappiness committed:
Adding the Initial Setup walkthrough to show user what to expect at f… (#1348)

* Adding the Initial Setup walkthrough to show user what to expect at first login.

* fix lint

* fix lint

* Revising & editing of new Initial Setup section

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

isFakeAccount committed:
Update package manager troubleshooting commands (#1347)

* Update package manager troubleshooting commands

Added command to clean APT repository metadata.

* Tweak flags & description of new apt list rm command

* Add new apt list rm command & apt clean description to Ubuntu article

This matches the previous commits' additions to the Pop!_OS article.

* Fix punctuation order for mid-sentence parenthetical at end of sentence

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

leviport committed:
Update Open Firmware/EC model table with current products (#1350)

Socialhappiness committed:
update pop basics article to align with 24.04 features (#1354)

* update pop basics article to align with 24.04 features

* fix lint

* fix (lint): Replace HTML tags with Markdown for italics

* fix (lint): Escape underscore in Pop!_OS

* fix (build): Remove extraneous leading slash from URL

* Revise updated Pop Basics article

- Made the YouTube embeds responsive (so they take up the full width available to them).
- Reworded some of the phrasing around the videos so they flow better with the rest of the content.
- Reworded the second intro sentence to avoid repeating the word `desktop` so many times in a row.
- Changed the `Pop!_OS COSMIC Desktop Environment` header to just `COSMIC Desktop Environment` to avoid confusion about COSMIC's name.
- Un-capitalized `desktop` in `desktop environment` and capitalized `Rust` as a programming language name.
- Added a hyperlink to the COSMIC webpage the first time COSMIC is mentioned.
- Reworded the Launcher's intro sentence.
- Added additional info about the Launcher's web search feature, and tweaked phrasing for the other features.
- Changed Dock wording to avoid confusion about applets vs. app icons (there was some incorrect information describing applets as "which applications are accessible with one click on the dock"; pinned apps are actually all part of a single applet).
- Moved mouse-based Workspaces info out of the shortcuts section into the main section above it, leaving only shortcuts info in the shortcuts section.
- Fixed inconsistencies in the capitalization of COSMIC and various COSMIC components.
- Reworded all generic "this" and "here" hyperlinks to describe what they're linking to for better accessibility and readability.
- Miscellaneous grammar fixes (commas, etc) and typo fixes (consiting, acces).

* fix (lint): Use 2-space indentation for ULs

* feat: Responsive video embeds

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

Socialhappiness committed:
fixed spelling

Socialhappiness committed:
fixed keyboard shortcut link to point to cosmic article

jacobgkau committed:
Add more info about toggling airplane mode

jacobgkau commented on content/bluetooth.md:
```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
 
 Bluetooth issues can be troubleshooted in several ways.  The first thing to check is toggling airplane mode which will sometimes get Bluetooth functioning again.  Next, make sure Bluetooth is enabled in the top bar, or in the <u>Bluetooth</u> system settings.
 
```
I've added more info on how to do this, including a screenshot of the toggle in the Wi-Fi dropdown menu.

jacobgkau committed:
Update Bluetooth enable screenshots to 24.04

jacobgkau committed:
Add basic usage section

jacobgkau committed:
Re-take some screenshots

BlakeRhodes committed:
Fix typo in configuration example (#1358)

Looks like a typo in the user group

Alexander-Kendrick committed:
Updated keystroke to reach systemd-boot

thomas-zimmerman reviewed (dismissed):
This is a really nice update to the article.

jacobgkau commented:
I wasn't finished working on this one, just needed to push a WIP commit in order to save my work so I could switch to a different branch this morning. I will finish the improvements needed for QA approval of this PR once I'm finished with the tech docs project I'm currently occupied with, and will address any remaining lint issues at that time as well. (Those consecutive blank lines were a placeholder for another screenshot.)

Alexander-Kendrick committed:
Merge pull request #1361 from system76/update-recovery

Updated keystroke to reach systemd-boot

Socialhappiness committed:
Removed the extensive backup and questioning section that was previously here, removed the older releases and troubleshooting sections to place in separate article because it was referencing a lot of unsupported releases. Added a few things to expect with cosmic with the article links. (#1359)

Co-authored-by: Levi Portenier <levi@system76.com>

HappyGelo05 committed:
New PR for Managing Apps Article (#1349)

* new PR for managing apps article

* managing-apps: Address some of the outstanding feedback

* managing-apps: Reorganize to place GUI first & old versions last, and remove out-of-scope sections

* managing-apps: Fix file names & image descriptions, remove unused images & optimize remaining ones

* managing-apps: Better organize Flatpak vs deb explanations

* managing-apps: Retake Repoman & CLI screenshots, add search screenshots

* fix (lint): Correct sub-bullet indentation

* fix (lint): Correct another sub-bullet indentation & surrounding line spacing

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

erinxocon committed:
Change the key name for the cache to force npm to reinstall everything and then rebuild

erinxocon committed:
Merge pull request #1362 from system76/invalidate-pipline-cache

Invalidate Pipeline Cache for troubleshooting

erinxocon committed:
debug print

erinxocon committed:
force install chrome for puppeteer

erinxocon committed:
(-) remove nuxt-dynamic-images

This is not in any actual vue code and is now causing a fatal error in the build process. It can safely be removed from the generation pipeline

erinxocon committed:
(*) revert some debug changes made in previous commits

erinxocon committed:
Merge pull request #1364 from system76/remove-nuxt-dynamic-image

Remove nuxt dynamic image

This is not in any actual vue code and is now causing a fatal error in the build process. It can safely be removed from the generation pipeline

jacobgkau committed:
Update favicon to higher-resolution version (#1366)

Socialhappiness committed:
routing the tech docs link directly to tech docs (#1365)

* routing the tech docs link directly to tech docs

* Address JS lint warnings

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

erinxocon committed:
(*) remove plausible and add google analytics (#1368)

Socialhappiness committed:
updated the largest size files I found in the docs and removed some t… (#1367)

* updated the largest size files I found in the docs and removed some text on the clear cache and cookies article while I replaced the gif with images.

* linked to files hosted on Google Drive and removed large files on support site

* fix: Markdown lint

* Simplify new guides wording

* Add Oxford comma

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

thomas-zimmerman committed:
Add section about COSMIC desktop and switchable graphics. (#1370)

* Add section about COSMIC desktop and switchable graphics.

---------

Co-authored-by: Emma Truong <emma@system76.com>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

bennetyee committed:
Update laptop-battery-thresholds.md (#1369)

* Update laptop-battery-thresholds.md

The firmware has been updated since Nov 2024 to save the threshold values to flash.

* Simplify wording & remove additional mention of limitation

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Thomas Zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

jacobgkau committed:
Replace incorrect Bluetooth version check

The content before would have displayed the version of the bluetoothctl utility,
not the supported Bluetooth version of the device. The new, simpler method shows
the actual Bluetooth version of the device.

jacobgkau committed:
Remove incorrect info on separate antennas for WiFi & BT

jacobgkau committed:
Revising & editing for accuracy & style

jacobgkau committed:
Better organization of additional commands

jacobgkau committed:
Update COSMIC screenshots to latest version & optimize PNGs

jacobgkau committed:
Convert PNGs to WebP

jacobgkau committed:
Consolidate audio tips

jacobgkau committed:
Tweak kernel modules heading

jacobgkau reviewed (approved):
Alright, this was a lot to sift through, but I think it's ready from my end.

- Updated all screenshots to COSMIC since this PR started them off on 22.04
- Removed outdated or incorrect info, some of which was already present before this PR (and some of which was introduced in the PR)
    - Two examples are a procedure for getting the supported Bluetooth version (that previously got the version of the software utility being used to check it instead), and an assertion that the two antennas on a wireless card are for Wi-Fi and Bluetooth separately (which is not generally the case).
- Further improved organization to try and increase the chances someone will jump to a section relevant to them vs. getting overwhelmed and just giving up (mainly trying to keep similar topics near each other & have more descriptive headings)

The images are using WebP because it seems our version of the ipx image module is too old to support AVIF. This update adds ~0.2MB to the repository.

This PR should be **Squashed and merged** when approved.
